report-generator/compile_latency.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+
+LATENCY_ADOC_FILE=$(readlink -f "include/latency-test-reports.adoc")
+STATS_FILE="include/stats_results"
+
+generate_latency_results()
+{
+    # Generate graph and stats associated to a publisher and subscriber test
+    # Parameters: $1: publisher result file
+    # $2: subscriber result file
+
+    local PUB_DATA_FILE=$1
+    local SUB_DATA_FILE=$2
+    python3 timestamp_analysis/timestamp_analysis.py \
+    $PUB_DATA_FILE \
+    $SUB_DATA_FILE >> $STATS_FILE
+}
+
+
+add_latency_results()
+{
+    # Generate and add latency tests results to final report
+    # Parameter: $1: path to tests results directory
+    echo "==  Latency tests results ==" >> "$LATENCY_ADOC_FILE"
```

Comment:
rm LATENCY_ADOC_FILE to avoid importing previous aborted tests

Reply:
This is already done, see the last lines of the script